cloudformation/custom_resource/cluster.yaml

```diff
@@ -0,0 +1,231 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: AWS ParallelCluster CloudFormation Template
+
+Parameters:
+
+  CustomLambdaRole:
+    Description: Custom role to use for PC Lambda
+    Type: String
+    Default: ''
+
+  AdditionalIamPolicies:
+    Description: Comma-delimited list of additional IAM Policies to add to the cluster (only used if CustomLambdaRole isn't provided).
+    Type: CommaDelimitedList
+    Default: ''
+
+Mappings:
+  ParallelCluster:
+    Constants:
+      Version: 3.6.0  # major.minor.patch+alpha/beta_identifier
+
+Conditions:
+  CustomRoleCondition: !Not [!Equals [!Ref CustomLambdaRole, '']]
+  UsePCPolicies: !Not [!Condition CustomRoleCondition ]
+  UseAdditionalIamPolicies: !Not [!Equals [!Join ['', !Ref AdditionalIamPolicies ], '']]
+
+Resources:
+  PclusterLayer:
+    Type: AWS::Lambda::LayerVersion
+    Properties:
+      LayerName: !Sub
+        - PCLayer-${StackIdSuffix}
+        - { StackIdSuffix: !Select [2, !Split ['/', !Ref 'AWS::StackId']] }
+      Description: Library which contains aws-parallelcluster python package and dependencies
+      Content:
+        S3Bucket: !Sub ${AWS::Region}-aws-parallelcluster
+        S3Key: !Sub
+        - parallelcluster/${Version}/layers/aws-parallelcluster/lambda-layer.zip
+        - { Version: !FindInMap [ParallelCluster, Constants, Version] }
+      CompatibleRuntimes:
+        - python3.9
```

Comment:
question, non-blocking: is this actually the only compatible runtime? Afaik 3.10 should work too
I guess we are using 3.9 here as we define the lambda with 3.9 right?